README.rst

```diff
@@ -155,13 +155,13 @@ At present, these rules can load the following versions of these tools:
 
 * :tool:`kustomize`
 
-  * `v4.5.1 <https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.1>`__ (default)
-  * `v4.4.1 <https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.4.1>`__
+  * `v4.5.1 <https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.3>`__ (default)
```

Comment:
```suggestion
  * `v4.5.3 <https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.3>`__ (default)
```
My assumption is the versions at beginning and end should match?

Reply:
Indeed they should. Thank you for catching that.